app/src/app.vue

```diff
@@ -99,6 +103,8 @@ useSystem();
 
 			<template #append>
 				<v-error :error="error" />
+				<br />
+				<v-button kind="danger" @click="reload()">{{ t('reload') }}</v-button>
```

Comment:
Drop the explicit call?

```suggestion
				<v-button kind="danger" @click="reload">{{ t('reload') }}</v-button>
```